src/components/manifold-subscription-details/manifold-subscription-details.tsx

```diff
@@ -4,6 +4,7 @@ import FixedFeature from 'components/shared/FixedFeature';
 import MeteredFeature from 'components/shared/MeteredFeature';
 import ConfigurableFeature from 'components/shared/ConfigurableFeature';
 import CostDisplay from 'components/shared/CostDisplay';
+import x from '@manifoldco/mercury/icons/x.svg';
```

Comment:
🤔 is there a better icon solution Mercury could deliver so that that’s easier to use? Such as path data?

Reply:
Yes! That would be nice! Then we could have an `Icon` component that works like this:
```jsx
<Icon icon={x}>
```

I recently was skimming this article: https://paulie.dev/posts/2020/04/svgIconSystems/

Reply:
Interesting—this seems to be how `@manifoldco/icons` does it. We started shipping whole `<svg>` elements in Mercury a) because the artboard size may differ, and b) it was just easier from Figma

But of course Nick always ships icons at a consistent size, so we can probably count on the `24 × 24` artboard. Given that, what should the import be? 

```ts
import { x } from '@manifold/mercury/icons';
```

(this would probably be an `index.ts` file in the `./icons` folder that TS would just infer)